doc/arkode/guide/source/Usage/SplittingStep/SplittingStepCoefficients.rst

```diff
@@ -0,0 +1,432 @@
+   :param order: A positive even number for the method order of accuracy.
+   :return: A :c:type:`SplittingStepCoefficients` structure if successful or a
+      ``NULL`` pointer if an argument was invalid or an allocation error
+      occurred.
+   
+   .. versionadded:: x.y.z
+
+
+.. c:function:: SplittingStepCoefficients SplittingStepCoefficients_SuzukiFractal(int partitions, int order)
+
+   Create the coefficients for the quintuple jump splitting method
+   :cite:p:`Suzuki:90`
+
+   .. math::
+      \begin{align*}
+      Q_h^{[2]} &= S_h, \\
+      Q_h^{[i+2]} &= Q_{\gamma_1 h}^{[i]} \circ Q_{\gamma_1 h}^{[i]} \circ
+      Q_{(1 - 4 \gamma_1) h}^i \circ Q_{\gamma_1 h}^{[i]} \circ
+      Q_{\gamma_1 h}^{[i]}, \\
+      y_n &= Q_h^{[order]}(y_{n-1}),
+      \end{align*}
+   
+   where :math:`S` is the Strang splitting and parameter :math:`\gamma_1` is
+   selected to increase the order by two each recursion.
+
+   :param partitions: The number :math:`P > 1` of partitions in the IVP.
+   :param order: A positive even number for the method order of accuracy.
+   :return: A :c:type:`SplittingStepCoefficients` structure if successful or a
+      ``NULL`` pointer if an argument was invalid or an allocation error
+      occurred.
+   
+   .. versionadded:: x.y.z
+
+
+.. c:function:: SplittingStepCoefficients SplittingStepCoefficients_Alloc(int sequential_methods, int stages, int partitions)
+
+   Allocates an empty SplittingStepCoefficients.
+
+   :param sequential_methods: The number :math:`r` of sequential methods
+      combined to produce the overall operator splitting solution.
+   :param stages: The number :math:`s` of stages.
+   :param partitions: The number :math:`P` of partitions in the IVP.
+   :return: A :c:type:`SplittingStepCoefficients` structure if successful or a
+      ``NULL`` pointer if an argument was invalid or an allocation error
+      occurred.
+   
+   .. versionadded:: x.y.z
+
+
+.. c:function:: SplittingStepCoefficients SplittingStepCoefficients_Create(int sequential_methods, int stages, int partitions, int order, sunrealtype* alpha, sunrealtype* beta)
+
+   Allocates a SplittingStepCoefficients and fills it with the given values.
+
+   :param sequential_methods: The number :math:`r` of sequential methods
+      combined to produce the overall operator splitting solution.
+   :param stages: The number :math:`s` of stages.
+   :param partitions: The number :math:`P` of partitions in the IVP.
+   :param order: The method order of accuracy.
+   :param alpha: An array of length ``sequential_methods`` containing the
+      weight of each sequential method used to produce the overall operator
+      splitting solution.
+   :param beta: An array of length
+      ``sequential_methods * (stages+1) * partitions`` containing the time nodes
+      of the inner integrations in the order
+
+      .. math::
+         \beta_{1,0,1}, \dots, \beta_{1,0,P}, \beta_{1,1,1}, \dots, \beta_{1,1,P}, \dots, \beta_{2,0,1}, \dots, \beta_{r,s,P}.
```

Comment:
It's either this or have `j = 1, ..., s+1` which also somewhat odd with respect to the end index. I like the current because an index of `j=0` indicates the start time, similar to how we use $t_0$ as the start of the time span.

Reply:
I think having all the indexing starting from 1 is a bit more clear even if it does not align as nicely with `t_0`